src/en/clowncorps/src/eu/kanade/tachiyomi/extension/en/clowncorps/ClownCorps.kt

```diff
@@ -0,0 +1,183 @@
+
+    private val creator = "Joe Chouinard"
+
+    // Text from: https://clowncorps.net/about/
+    private val synopsis = "Clown Corps is a comic about crime-fighting clowns. " +
+        "It's pronounced \"core.\" Like marine corps."
+
+    override val client = network.client.newBuilder()
+        .addInterceptor(TextInterceptor())
+        .build()
+
+    override fun fetchPopularManga(page: Int): Observable<MangasPage> = Observable.fromCallable {
+        MangasPage(
+            listOf(
+                SManga.create().apply {
+                    title = name
+                    artist = creator
+                    author = creator
+                    description = synopsis
+                    status = SManga.ONGOING
+                    // Image from: https://clowncorps.net/about/
+                    thumbnail_url = "https://clowncorps.net/wp-content/uploads/2022/11/clowns41.jpg"
+                    setUrlWithoutDomain("/comic")
+                },
+            ),
+            hasNextPage = false,
+        )
+    }
+
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList) =
+        fetchPopularManga(page)
+
+    override fun fetchMangaDetails(manga: SManga): Observable<SManga> =
+        Observable.just(manga.apply { initialized = true })
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val pageStatus = response.asJsoup().select("#paginav li.paginav-pages").text()
+        val pageCount = pageStatus.split(" ").last().toInt()
+
+        val chapters = mutableListOf<SChapter>()
+
+        for (page in 1..pageCount) {
+            val url = "https://clowncorps.net/comic/page/$page/"
+            val resp = client.newCall(GET(url, headers)).execute()
+            if (resp.code == 404) break
+            val comics = resp.asJsoup().select(".comic")
+            for (comic in comics) {
+                val link = comic.selectFirst(".post-title a")?.attr("href")
+                val title = comic.selectFirst(".post-title a")?.text()
+                val postDate = comic.selectFirst(".post-date")?.text()
+                val postTime = comic.selectFirst(".post-time")?.text()
+                if (link == null || title == null || postDate == null || postTime == null) continue
+                val date = parseDate("$postDate $postTime")
+
+                val chapter = SChapter.create().apply {
+                    setUrlWithoutDomain(link)
+                    name = title
+                    date_upload = date
+                }
+                chapters.add(chapter)
+            }
+        }
+
+        // Add chapter numbers to the chapters, to ensure correct sorting.
+        // Seems to be unnecessary, as it already bases the order on the chapter names.
+        // Kept here, commented out, just in case.
+        /*chapters.forEachIndexed { index, sChapter ->
+            sChapter.chapter_number = chapters.size - index.toFloat()
+        }*/
+
```

Comment:
```suggestion
```
remove
also does it work without it?

Reply:
It works with it commented out, yes. I've been using it to read the comic for the past few days without any issues.
But I wanted to keep this snippet around in case the automatic sorting happens to break at some point.
It's not hurting anything by being there, is it?

Reply:
This snippet forces chapter number to align with "sort by source order" in the app, so it's completely useless.

Reply:
But sure, I can remove it completely, if you prefer.
I must admit I don't see much point in discussing a small commented out piece of code.